OneCommunityGlobal/HighestGoodNetworkApp
# Xiao F update teams page sorting part b

# Description
![CleanShot 2023-12-29 at 16 36 28](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/72a27e0c-5c5b-48b9-9e7c-17dc255038e9)


## Related PRS (if any):
To test this frontend PR you need to checkout the [#671](https://github.com/OneCommunityGlobal/HGNRest/pull/671) backend PR.

## Main changes explained:
- Update file `src/components/Teams/Teams.jsx` for default sorting by modifiedDatetime

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner/admin user (Right now, you need an owner account to toggle 'Active' status for a team)
5. go to Other Links-> Teams-> Toggle 'Active' for a team or add/remove member from a team -> Refresh page -> The team modified will appear as the first one

## Screenshots or videos of changes:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/338c72f4-8584-4a54-8c97-416d184cc63b
